packages/react/docs/render.md

```diff
@@ -0,0 +1,23 @@
+# render.tsx
+
+#render
+
+<!--meta:render
+Unsupported: ":render"
+-->
+
+Bar and [Column](./Column.md) are effectively the same class, but have one different default value - their _orientation_. They support all of the same properties.
```

Comment:
Did you mean to use the chart_Bar text here?

Reply:
Nope